app/models/stripe_account.rb

```diff
@@ -0,0 +1,16 @@
+class StripeAccount < ActiveRecord::Base
+  belongs_to :enterprise
+  validates :stripe_user_id, :stripe_publishable_key, presence: true
+  validates :enterprise_id, uniqueness: true
```

Comment:
we must enforce this constrain at db level as well.

Reply:
Resolved